src/components/structures/MatrixChat.js

```diff
@@ -227,8 +229,6 @@ export default React.createClass({
     componentWillMount: function() {
         SdkConfig.put(this.props.config);
 
-        if (!SettingsStore.getValue("analyticsOptOut")) Analytics.enable();
```

Comment:
Are we deliberately re-prompting people who've already opted out? Might be nice to propagate the old `analyticsOptOut` to `analyticsOptIn`.

Reply:
Ah, fair enough.